skills/mobile-use-setup/references/platform-vs-local.md

```
@@ -0,0 +1,138 @@
+
+### Cons
+- Requires internet connection
+- Dependent on Minitap service
+- Less control over model selection
+
+### Setup
+
+1. Sign up at https://platform.minitap.ai
+2. Create API key
+3. Add to `.env`:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)